daemon/containerfs_linux.go

```diff
@@ -0,0 +1,221 @@
+package daemon // import "github.com/docker/docker/daemon"
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
```

Comment:
This changes made go-multierror a direct dependency (no longer indirect only); looks like CI didn't pick that up (wondering if we could have a way to "fast check" such changes 🤔); opened https://github.com/moby/moby/pull/44453 to fix that.